straxen/online_monitor.py

```diff
@@ -0,0 +1,35 @@
+from straxen.rundb import get_mongo_url, default_mongo_dbname

```

Comment:
Ok, here we're importing `get_mongo_url`, but neither of these quantities get exported from `rundb.py`

Reply:
You don't need to export it. The @export allows you to e.g. do:
```straxen.DAQReader`` 
instead of
``straxen.plugins.daqreader.DAQReader`` 

But either works. For these functions I say there is no need for e.g. a `` straxen.get_mongo_url.``